integration-tests/chopsticks/src/tests/xcm/assetHub/limitedReserveTransferSpiritnetAssetHub.test.ts

```diff
@@ -0,0 +1,57 @@
+import { test } from 'vitest'
+import { sendTransaction, withExpect } from '@acala-network/chopsticks-testing'
+
+import * as SpiritnetConfig from '../../../network/spiritnet.js'
+import * as AssetHubConfig from '../../../network/assetHub.js'
+import { KILT, initialBalanceKILT, keysAlice } from '../../../utils.js'
+import { spiritnetContext, assetHubContext, getFreeBalanceSpiritnet } from '../../index.js'
+import { getAccountLocationV3, getNativeAssetIdLocation, getSiblingLocation } from '../../../network/utils.js'
+import { checkBalance, createBlock, hexAddress, setStorage } from '../../utils.js'
+
+test('Limited Reserve Transfers from Spiritnet Account Alice -> AH Account Alice', async ({ expect }) => {
+	const { checkEvents, checkSystemEvents } = withExpect(expect)
+
+	// Assign alice some KILT tokens
+	await setStorage(spiritnetContext, {
+		...SpiritnetConfig.assignNativeTokensToAccounts([keysAlice.address], initialBalanceKILT),
+		...SpiritnetConfig.setSafeXcmVersion3(),
+	})
+
+	// Balance of the AH sovereign account before the transfer
+	const assetHubSovereignAccountBalance = await getFreeBalanceSpiritnet(AssetHubConfig.siblingSovereignAccount)
+
+	// check initial balance of Alice on Spiritnet
+	await checkBalance(getFreeBalanceSpiritnet, keysAlice.address, expect, initialBalanceKILT)
+
+	const aliceAddress = hexAddress(keysAlice.address)
+	const assetHubDestination = { V3: getSiblingLocation(AssetHubConfig.paraId) }
+	const beneficiary = getAccountLocationV3(aliceAddress)
+	const kiltAsset = { V3: [getNativeAssetIdLocation(KILT)] }
+
+	const signedTx = spiritnetContext.api.tx.polkadotXcm
+		.limitedReserveTransferAssets(assetHubDestination, beneficiary, kiltAsset, 0, 'Unlimited')
+		.signAsync(keysAlice)
+
+	const events = await sendTransaction(signedTx)
+
+	// Check sender state
+	await createBlock(spiritnetContext)
+
+	// Check events sender
+	checkEvents(events, 'xcmpQueue').toMatchSnapshot('sender events xcm queue pallet')
```

Comment:
It would be nice to also rename this part of the snapshots, if possible, with something a bit more... descriptive, let's say...

Reply:
Perhaps defining a structure and sticking to it. Something like `<pallet_name>::events` for events. I definitely open to better proposals, but the current naming is not very descriptive.

Reply:
What you think of this here: [7170932](https://github.com/KILTprotocol/kilt-node/pull/668/commits/7170932d310e9f1d38f0107b0d53691adebe06ae) 

its `[sender/receiver]::pallet::[events] asset x`

Reply:
Nice! It works, for now!